### PR TITLE
aacgain: fix src

### PR DESCRIPTION
**Pull request by RaghavSood**
###### Motivation for this change

#104048

The project moved to gitlab

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by ryantm — APPROVED** [review]

File: pkgs/applications/audio/aacgain/default.nix [hunk 1]
```diff
@@ -1,9 +1,9 @@
-{ stdenv, fetchFromGitHub, fetchpatch }:
+{ stdenv, fetchFromGitLab, fetchpatch }:
 
 stdenv.mkDerivation {
   name = "aacgain-1.9.0";
```
Comment (prusnak) [inline pkgs/applications/audio/aacgain/default.nix:4]:
```suggestion
stdenv.mkDerivation rec {
  pname = "aacgain";
  version = "1.9.0";
```

File: pkgs/applications/audio/aacgain/default.nix [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-  src = fetchFromGitHub {
+  src = fetchFromGitLab {
     owner = "mulx";
     repo = "aacgain";
```
Comment (prusnak) [inline pkgs/applications/audio/aacgain/default.nix:8]:
```suggestion
    repo = pname;
```

**Review by prusnak — COMMENTED** [review]
LGTM, but left two optional suggestions

**Comment by RaghavSood** [comment]
@prusnak Agreed on the suggestions, but I believe they are already being discussed in #104045 - wanted to scope this PR to solve the major issue on hand

**Pull request merged**